src/components/core/Drawer.vue

```diff
@@ -136,7 +161,17 @@ export default {
       } else {
         this.responsive = false
       }
-    }
+    },
+    isView (bool) {
+    // return links to views for true argument and non-views for false argument
+      if (bool) {
+        return this.links.filter(function(u) {
+          return u.view;
+      })} else {
+        return this.links.filter(function(u) {
+          return !u.view;
+      })}
+    },
```

Comment:
I think it could be simplified a bit?

```javascript
    isView (bool) {
    // return links to views for true argument and non-views for false argument
      return this.links.filter((u) => {
        return u.view === bool
      })
    },
```

Reply:
Nice suggestion, I hope so! I did suspect what I had coded to filter the views could be, but I have little experience in JavaScript so I was not sure quite how to, consolidate. I'll give this a try & squash it in if it (or something similar) works.